### ISSUE TITLE
Class attributes `__qualname__` and `__module__` should be accessible within class body

### ISSUE THREAD

**Issue by erictraut** [python/mypy#10570]
The attributes `__qualname__` and `__module__` appear to be accessible from within a class body. These accesses don't generate a runtime error, but mypy reports a "not defined" error.

```python
class Foo:
    print(__doc__)
    print(__name__)
    print(__qualname__)  # error: Name '__qualname__' is not defined
    print(__module__)  # error: Name '__module__' is not defined
```

This is related to the following bug filed against pyright: https://github.com/microsoft/pylance-release/issues/1376.

**Comment by supreetsingh10** [comment]
I would like to work on this one, where should I start out with this problem?

**Comment by jli** [comment]
I think the title of this should be this, right?

> Class attributes `__qualname__` and `__module__` should be accessible within class body

**Comment by erictraut** [comment]
@jli, yes, good catch. I've updated the issue name.
